vscode/src/telemetry.ts

```diff
@@ -250,3 +254,19 @@ export function sendTelemetryEvent<E extends keyof EventTypes>(
     )}`,
   );
 }
+
+function getBrowserRelease(): string {
+  if (navigator.userAgentData) {
+    const browser =
+      navigator.userAgentData?.brands[
```

Comment:
There's no point adding this here, you've already checked `navigator.userAgentData' is not 'falsy' in the if check. The if check is where `navigator.userAgentData?.brands` should be used, which is a safe way to check userAgentData.brands exists on navigator before you get to this line.